ntbs-ui-tests/Features/TransferNotificationCookieOverride.feature

```diff
@@ -0,0 +1,59 @@
+@CookieOverride
```

Comment:
This is a new file because the backgrounds between this and TransferNotification.feature are different, and unfortunately you can't have have two backgrounds in the same file and have tags pick which one to run

Reply:
What's different about the backgrounds?

Reply:
The difference was the 'And I select Leeds UITester for 'TransferRequest_CaseManagerId' step, which I've now updated the method for so I have removed the duplication of the backgrounds